stable_baselines/common/evaluation.py

```diff
@@ -1,17 +1,31 @@
+import typing
+from typing import Callable, List, Optional, Tuple, Union
+
+import gym
 import numpy as np
 
 from stable_baselines.common.vec_env import VecEnv
 
+if typing.TYPE_CHECKING:
+    from stable_baselines.common.base_class import BaseRLModel
```

Comment:
Is importing necessary when using strings for types? This is nitpicking, but seems rather wonky import and I'd rather avoid such in the future with sb3 if in any way possible ^^

Reply:
hmm, good question, will try without

Reply:
but how would the type checker know which type your are referring to if it is not imported?

Reply:
Yeah I was not sure if it wants to properly check that the classes/types exist. Maybe this could be just standard import so we avoid mental load of "if being typed, do this extra stuff"?

Reply:
"line 22, in <module>: Name 'BaseRLModel' is not defined [name-error]"

>Edit: Ach, but then it will be "Unused import"...

That's why I usually use `# noqa: F401` in that case


this type of import is only required to prevent circular import, so it should not happened that often (mostly when you import base classes).

Reply:
Ok, if this won't become a habit then exception in this case is alright.